Traceback (most recent call last):
  File "/fh/fast/matsen_e/whowards/usher-clade-reconstructions/historydag/scripts/agg_mut.py", line 1973, in <module>
    cli()
  File "/home/whowards/anaconda3/envs/build_usher_trees/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/whowards/anaconda3/envs/build_usher_trees/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/whowards/anaconda3/envs/build_usher_trees/lib/python3.9/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/whowards/anaconda3/envs/build_usher_trees/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/whowards/anaconda3/envs/build_usher_trees/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/fh/fast/matsen_e/whowards/usher-clade-reconstructions/historydag/scripts/agg_mut.py", line 1958, in select_clades
    height2ids.append(id)
AttributeError: 'dict' object has no attribute 'append'
